pom.xml

```diff
@@ -81,32 +84,32 @@
         <dependency>
             <groupId>io.confluent</groupId>
             <artifactId>kafka-connect-storage-common</artifactId>
-            <version>${confluent.version}</version>
+            <version>${project.version}</version>
```

Comment:
IIUC this means these deps are following the HDFS version, but `kafka-connect-storage-common` isn't guaranteed to be the same version as HDFS, no? shouldn't this be the parent version which is `kafka-connect-storage-common`?

Reply:
they go in lockstep for now. I'll update when I cut the release in storage-common